xml/System.IO.Pipes/PipeOptions.xml

```diff
@@ -102,7 +102,7 @@
       </ReturnValue>
       <MemberValue>536870912</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that the pipe has the same elevation level as the user that created it, and is owned and can only be used by the user that created it.</summary>
```

Comment:
I was going to suggest, "When used to create a named pipe server, causes it to only connect to a client created by the same user. When used to create a named pipe client, causes it to only connect to a server created by the same user. On Windows, it verifies both the user account and elevation level."

But I guess what you have basically says the same thing.

Reply:
Thank you @danmosemsft for the suggestion. How does this sound to you?:
```suggestion
        <summary>When used to create a <see cref="T:System.IO.Pipes.NamedPipeServerStream" /> instance, indicates that the pipe can only only be connected to a client created by the same user. When used to create a <see cref="T:System.IO.Pipes.NamedPipeClientStream" /> instance, indicates that the pipe can only connect to a server created by the same user. On Windows, it verifies both the user account and elevation level.</summary>
```